packages/react-components/src/components/utils/useLongPress.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { useRef, useState } from 'react';
+
+export default function useLongPress(onClick: () => void, onLongPress: () => void) {
```

Comment:
Does this work for keyboard and ally users?

Reply:
Updated longpress hook to include keyboard event